packages/main/src/TimePicker.hbs

```diff
@@ -0,0 +1,73 @@
+<div id="{{this._id}}" formatPattern="{{formatPattern}}" class="ui5-timepicker">
```

Comment:
This should be "ui5-timepicker-root", when the component name and the class name are the same, the styles in Internet Explorer breaks.

Reply:
done